service/src/main/java/bio/terra/workspace/amalgam/landingzone/azure/LandingZoneApiController.java

```diff
@@ -37,16 +39,20 @@ public LandingZoneApiController(
   @Override
   public ResponseEntity<ApiAzureLandingZoneResult> createAzureLandingZone(
       @RequestBody ApiCreateAzureLandingZoneRequestBody body) {
+    AuthenticatedUserRequest userRequest = getAuthenticatedInfo();
```

Comment:
Suggest using the new and improved `BearerToken` and `SamUser` classes that Doug added rather than `AuthenticatedUserRequest`. You can save calls to Sam by using those.